tests/pcr_selections_tests.rs

```diff
@@ -164,4 +169,100 @@ mod test_pcr_selection_builder {
         // The try_from should then fail.
         PcrSelections::try_from(tpml_pcr_selection).unwrap_err();
     }
+
+    #[test]
```

Comment:
It would be nice to add a test for `difference` where the two operators have multiple `HashingAlgorithm` keys!

Reply:
fixed